queries.sql

```diff
@@ -64,4 +64,25 @@ SELECT neutered, SUM(escape_attempts) AS total_escapes FROM animals GROUP BY neu
 SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animals GROUP BY species;
 
 -- Average number of escape attempts per animal type of those born between 1990 and 2000
-SELECT species, AVG(escape_attempts) AS avg_num_of_escape FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) AS avg_num_of_escape FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+-- What animals belong to Melody Pond?
+SELECT a.name FROM animals a JOIN owners b ON a.owner_id = b.id WHERE b.full_name = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT a.name FROM animals a JOIN species s ON a.species_id = s.id WHERE s.name = 'Pokemon';
```

Comment:
You did well with your project, kindly note that the type of JOIN used here should be `LEFT JOIN` , please implement this so that your database query would be accurate 😄